### PR TITLE
Happy 2018 /tg/station and coderbus

### PR DESCRIPTION
**Pull request by optimumtact**
We will be resetting PRB for the new year and probably we should run the competition for worst change of 2017 as well

Can I just say as well what a fantastic year for us here at the codebase, components, datum antags (thanks antur), lots of new high quality contributors and some maintainers, a new headcoder and a whole boat load of fixes and improvements. You've all done a great job and should be proud of what you achieved. Even MRTY

### CODE REVIEW

**Comment by praisenarsie** [comment]
🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉

**Comment by MoreRobustThanYou** [comment]
it's still fucking 2017

no fuck off with your timezones, EST is the one true timezone

**Comment by optimumtact** [comment]
MRTY's doing well isrevoked for being a grinch

**Comment by MoreRobustThanYou** [comment]
fuck you citrus fruit, may you forever be condemned to be my kitten's playtoy

**Comment by outbools** [comment]
citruses are superior beings be careful with your wording

**Comment by Cyberboss** [comment]
We'll also be holding the vote for the biggest fuckup of the year

**Comment by vuonojenmustaturska** [comment]
@Cyberboss there's no way it isn't #25913 though

**Comment by FantasticFwoosh** [comment]
I hope to submit more code for 2018 & really make it my year.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -22,20 +22,13 @@ There is an open list of approachable issues for [your inspiration here](https:/
...
-
-Project Managers are responsible for recruiting and firing maintainers, enforcing coding standards, and reverting changes that should have not been committed. Project Managers are assigned by Project Leads. On things that Project Managers disagree on they are to refer to the Project Leads for advice. It is encouraged that if you do not want to waste time working on a feature, that might be denied, that you ask a Project Manager first.
+Don't fuck with them they're straight up killers.
 
 **Maintainers**
 
-Maintainers are quality control. If a proposed pull request does not meet the mentioned quality specifications then it can be closed if you fail to satisfy them. Maintainers are required to give a reason for closing the pull request.
-
-Maintainers can revert your changes if they feel they are not worth maintaining or if they did not live up to the quality specifications.
+Maintainers will argue about your pr in their secret irc channel and then it will be merged and then reverted in some kind of metawar you can barely comprehend, you can attempt to improve this process by using the 8b command in their channel to convince them
```
Comment (duncathan) [inline .github/CONTRIBUTING.md]: for the record many maintainers hardly comprehend the metawar either, for all you plebs out there

**Comment by Armhulen** [comment]
@optimumtact happy hanukkah

**Comment by Jalleo** [comment]
I dont care about PRB anyhow I have never made a feature that has been merged. Because I just cant code for that.

**Comment by Armhulen** [comment]
why?

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (FantasticFwoosh) [reply .github/CONTRIBUTING.md]: It's easy really, maintainers respond with chemical pheromones to PR's of either fear or desire to communicate with the rest of the lower-caste coder drones.

**Comment by ninjanomnom** [comment]
I only started coding for ss13 with much frequency this year and it's been a fun ride going from barely knowing how to find a var to, given some effort, understanding any part of the codebase that I feel like. It's been especially gratifying to move on to helping others with that knowledge and grow others up from the same starting point.

See you all around in the new year and though we may have wildly differing opinions on where the gameplay should move, I love and respect you all for taking the time to contribute to this project ^.^

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (ninjanomnom) [reply .github/CONTRIBUTING.md]: There is no war in ~~ba sing se~~ coderbus

**Comment by KorPhaeron** [comment]
my head is still fuzzy from being sick and I can't think of anything very profound to say but you're all heroes for keeping this nonsense going another year thank you all

**Comment by JJRcop** [comment]
So yeah yeah codebase/server separation but if the servers went down and nobody took up the mantle eventually all the coders would disperse to other stations

Somehow this ship still sails

**Comment by optimumtact** [comment]
I don't see how this is relevant

**Comment by ninjanomnom** [comment]
We'd just have new servers pop up using the same code

**Comment by BeeSting12** [comment]
I love all of you for making this possible even oranges

**Pull request closed without merging**